src/apig_wsgi.py

```diff
@@ -68,6 +69,7 @@ def get_environ(event, context, binary_support):
         "wsgi.multithread": False,
         "wsgi.run_once": False,
         "wsgi.version": (1, 0),
+        "wsgi.url_scheme": "http",
```

Comment:
So accordnig to the [WSGI spec](https://www.python.org/dev/peps/pep-3333/), apig-wsgi should have always been sending this. This is a good spot.

Hardcoding to "http" or "https" is wrong though. I have a sample API Gateway deployment, which seems to only communicate the protocol in the  `X-Forwarded-Proto` header. Is this also the case on Lambda-on-ELB?

Reply:
All seems to work if `wsgi.url_scheme` exist and is empty

Reply:
That might make your health check pass but break other WSGI frameworks. The WSGI spec basically says it should be `http` or `https` only.

Reply:
Right, so in a normal request, from my tests, you always have the x-forwarded-proto, so the "http" would just be a placeholder.   

So leave it to http?  i don't think we have much choice if it's in the spec.

Reply:
Right - I missed that. I think you're right, we should be leaving it as `http`.